pkg/lib/cockpit.js

```diff
@@ -3794,8 +3794,12 @@ function factory() {
         if (po_data) {
             const translated = po_data[key];
             if (translated?.[1])
-                return translated[1];
+                string = translated[1];
         }
+
+        if (window.localStorage.test_l10n === 'true')
```

Comment:
This is a hot codepath. Could this be read once and then cached? You'd then need a reload of the page after setting the storage flag, but I think that's acceptable.